packages/account/Readme.md

```diff
@@ -0,0 +1,133 @@
+| entryPointAddress  | Since entrypoint can have different addresses you can call getSupportedEntryPoints() on bundler instance for supported addresses list|
+
+```typescript
+// This is how you create bundler instance in your dapp's
+import { IBundler, Bundler } from '@biconomy/bundler'
+import { ChainId } from "@biconomy/core-types";
+
+const bundler: IBundler = new Bundler({
+    bundlerUrl: '',      
+    chainId: ChainId.POLYGON_MAINNET,
+    entryPointAddress: '',
+  })
+```
+
+### Paymaster
+
+ERC4337, Account abstraction, introduces the concept of Paymasters. These specialised entities play a pivotal role in revolutionising the traditional gas payment system in EVM transactions. Paymasters, acting as third-party intermediaries, possess the capability to sponsor gas fees for an account, provided specific predefined conditions are satisfied.
+
+**paymasterUrl** you can get this value from biconomy dashboard. https://dashboard.biconomy.io
+
+```typescript
+// This is how you create paymaster instance in your dapp's
+import { IPaymaster, BiconomyPaymaster } from '@biconomy/paymaster'
+
+  const paymaster: IPaymaster = new BiconomyPaymaster({
+    paymasterUrl: '' // you can get this value from biconomy dashboard. https://dashboard.biconomy.io
+  })
+```
+
+## Smart Account instance configuration
+
+| Key           | Description |
+| ------------- | ------------- |
+| signer        | This signer will be used for signing userOps for any transactions you build. You can supply your your EOA wallet signer|
+| chainId       | This represents the network your smart wallet transactions will be conducted on. Take a look following Link for supported chain id's |
+| rpcUrl        | This represents the EVM node RPC URL you'll interact with, adjustable according to your needs. We recommend to use some private node url for efficient userOp building|
+| paymaster     | you can pass same paymaster instance that you have build in previous step. Alternatively, you can skip this if you are not interested in sponsoring transaction using paymaster|
+|               | Note: if you don't pass the paymaster instance, your smart account will need funds to pay for transaction fees.|
+| bundler       | You can pass same bundler instance that you have build in previous step. Alternatively, you can skip this if you are only interested in building userOP|
+
+
+## Example Script Usage
+
+```typescript
+// This is how you create BiconomySmartAccount instance in your dapp's
+
+import { BiconomySmartAccount, BiconomySmartAccountConfig } from "@biconomy/account"
+import { IHybridPaymaster, PaymasterMode, SponsorUserOperationDto, PaymasterFeeQuote } from '@biconomy/paymaster'
+
+const biconomySmartAccountConfig: BiconomySmartAccountConfig = {
+    signer: wallet.getSigner(),
+    chainId: ChainId.POLYGON_MAINNET, 
+    rpcUrl: '',
+    paymaster: paymaster, //you can skip paymaster instance if not interested in transaction sposoring
```

Comment:
for how to create paymaster instance, refer to paymaster docs.

one can also create their custom paymaster as long as it implements IPaymaster interface. this should be mentioned here/somewhere

Reply:
I have already added the snippet to create a Paymaster instance before this step